src/components/Layout/Layout.js

```diff
@@ -20,25 +20,40 @@ const Layout = ({ children, isGlow, headerIsWhite }) => {
   const mainClasses = classNames(styles.main, styles.hidden);
   const footerClasses = classNames(styles.footer, styles.hidden);
 
+  const childRef = useRef(null);
+
+  const [remBoxShadow, useRemBoxShadow] = React.useState(null);
+
+  function removeBoxShadow(MailUs) {
```

Comment:
can you move this function to the MailUs component? 

would it work if you will do
`<BoxShadowContext.Provider value={{ useRemBoxShadow }}>` and work with `useRemBoxShadow` inside <MailUs>?

Reply:
`function removeBoxShadow(MailUs) { `

MailUs - naming

Reply:
MailUsRef